climpred/tests/test_perfect_model_prediction.py

```diff
@@ -3,141 +3,62 @@
 
 from climpred.bootstrap import bootstrap_perfect_model
 from climpred.loadutils import open_dataset
-from climpred.prediction import compute_perfect_model, compute_persistence_pm
+from climpred.prediction import (all_metric_strings, compute_perfect_model,
+                                 compute_persistence)
 
-xskillscore_metrics = ('pearson_r', 'rmse', 'mse', 'mae')
-xskillscore_distance_metrics = ('rmse', 'mse', 'mae')
-PM_only_metrics = ('nrmse', 'nmse', 'nmae')
-PM_comparisons = ('e2c', 'm2c', 'm2e', 'm2m')
-all_metrics = xskillscore_metrics + PM_only_metrics
+PM_comparisons = ('e2c', 'm2c', 'm2e', 'm2m')[:2]
```

Comment:
only take the two first and fast comparisons. much faster now the testing

Reply:
What if there is an issue with `m2e` or `m2m`? We wouldn't catch it. Maybe there's a smaller example to test them on to see that they work?

Reply:
oh just that they take so long. Can and should put them back in